doc/man3/EVP_EncryptInit.pod

```diff
@@ -274,6 +275,10 @@ is not supported.
 EVP_CIPHER_CTX_ctrl() allows various cipher specific parameters to be determined
 and set.
 
+EVP_CIPHER_CTX_rand_key() generates a random key of the appropriate length based
+on the cipher context. The EVP_CIPHER can provide its own random key generation
```

Comment:
should add a sentence saying that key should b at least as large as the value returned by evp-xxxx.

Reply:
Reformulated. Feel free to propose the final wording, if its not okay .